assets/src/edit-story/app/api/testUtils/story.js

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
We've been doing `test/_utils.js` for testing helpers.

Reply:
We've [explicitly excluded `_utils`](https://github.com/google/web-stories-wp/blob/master/tests/js/jest.config.js#L44) from the test runner for this purpose.

Reply:
Done.